backend/app/db/alembic_test/versions/e19876d7c6c7_make_difficulty_nullable.py

```
@@ -0,0 +1,109 @@
+    op.alter_column('files', 'user_id',
+               existing_type=sa.String(length=128),
+               type_=mysql.INTEGER(),
+               existing_nullable=False)
+    op.drop_column('files', 'updated_at')
+    op.drop_column('exercises', 'difficulty')
+    op.drop_column('exercises', 'title')
+    op.create_table('users',
+    sa.Column('id', mysql.INTEGER(), autoincrement=True, nullable=False),
+    sa.Column('username', mysql.VARCHAR(collation='utf8mb4_unicode_ci', length=256), nullable=False),
+    sa.Column('email', mysql.VARCHAR(collation='utf8mb4_unicode_ci', length=256), nullable=False),
+    sa.Column('password', mysql.VARCHAR(collation='utf8mb4_unicode_ci', length=256), nullable=False),
```

Ruff (0.8.2)

Line too long (101 > 100)

(E501)